UI Development/gui.pde

```diff
@@ -14,6 +14,12 @@ class Gui {
   String lines[];
   String splitLine[];
   String linesSplit[][];
+    
+  Airport airport;
+  Date date;
+  Delays delay;
+  ChartPage charts = new ChartPage();
+  Scroll scroll = new Scroll();
```

Comment:
For these declarations, see the comment made on the encapsulation of our program.